test/helpers/utils.go

```diff
@@ -85,6 +85,23 @@ func RenderTemplate(tmplt string) (*bytes.Buffer, error) {
 	return content, nil
 }
 
+func RenderTemplateWithData(sourcePath string, destPath string, data interface{}) error {
```

Comment:
`test/k8s/bgp.go` is using templates as well. It would be good to have a sep. commit to convert the usage in that file to what's being introduced here. Nice addition btw :+1: